spec/lib/pretty_text_spec.rb

```diff
@@ -662,27 +659,27 @@ def cook(*args)
         "<pre><code class='lang-auto'>\n</code></pre>",
       )
       expect(PrettyText.cook("```ruby startline=3 $%@#\n```")).to match_html(
-        "<pre data-code-startline='3'><code class='lang-ruby'>\n</code></pre>",
+        "<pre data-code-startline='3' data-code-wrap='ruby'><code class='lang-ruby'>\n</code></pre>",
       )
       expect(PrettyText.cook("```mermaid a_-你=17\n```")).to match_html(
-        "<pre data-code-a_-='17' data-code-wrap='mermaid'><code class='lang-plaintext'>\n</code></pre>",
+        "<pre data-code-a_-='17' data-code-wrap='mermaid'><code class='lang-mermaid'>\n</code></pre>",
       )
       expect(
         PrettyText.cook("```mermaid foo=<script>alert(document.cookie)</script>\n```"),
       ).to match_html(
-        "<pre data-code-foo='&lt;script&gt;alert(document.cookie)&lt;/script&gt;' data-code-wrap='mermaid'><code class='lang-plaintext'>\n</code></pre>",
+        "<pre data-code-foo='&lt;script&gt;alert(document.cookie)&lt;/script&gt;' data-code-wrap='mermaid'><code class='lang-mermaid'>\n</code></pre>",
       )
       expect(PrettyText.cook("```mermaid foo=‮ begin admin o\n```")).to match_html(
-        "<pre data-code-wrap='mermaid'><code class='lang-plaintext'>\n</code></pre>",
+        "<pre data-code-wrap='mermaid'><code class='lang-mermaid'>\n</code></pre>",
```

Comment:
> This file contains bidirectional Unicode text that may be interpreted or compiled differently than what appears below. To review, open the file in an editor that reveals hidden Unicode characters.

just wondering – do you think that one weird character here on purpose?

Reply:
I think it is, yes. I've added a comment and updated it to use explicit `\uXXXX` syntax instead of including the real unicode character in the ruby code.